ceph-volume-ansible-prs/build/build

```diff
@@ -30,6 +30,8 @@ elif [[ "$ghprbTargetBranch" == "nautilus" ]]; then
     CEPH_ANSIBLE_BRANCH="stable-4.0"
 elif [[ "$ghprbTargetBranch" == "octopus" ]]; then
     CEPH_ANSIBLE_BRANCH="stable-5.0"
+elif [[ "$ghprbTargetBranch" == "pacific" ]]; then
+    CEPH_ANSIBLE_BRANCH="stable-6.0"
```

Comment:
There's no `stable-6.0` branch at this time. v6.0.0alpha tags are currently on the `master` branch.

Reply:
Should I leave this off until then?  Or merge it now?

Reply:
I'll use the alpha branch name